src-js/fontra-core/src/glyph-data.js

```diff
@@ -111,3 +116,55 @@ export function getGlyphInfoFromGlyphName(glyphName) {
   parseGlyphDataCSV();
   return glyphDataByName.get(glyphName);
 }
+
+export function guessGlyphPlaceholderString(codePoints, glyphName) {
+  let glyphString = "";
+  let direction = undefined;
+
+  if (codePoints?.[0]) {
+    glyphString = getCharFromCodePoint(codePoints[0]);
+  }
+
+  if (!glyphString && glyphName) {
+    const [baseGlyphName, extension] = splitGlyphNameExtension(glyphName);
+
+    let baseGlyphNames = [baseGlyphName];
+    if (baseGlyphName.indexOf("_") != -1) {
+      const [base, suffix] = splitGlyphNameExtension(baseGlyphName, "-");
+      baseGlyphNames = base.split("_").map((name) => name + suffix);
+    }
+
+    const codePoints = baseGlyphNames
+      .map((name) => getCodePointFromGlyphName(name))
+      .filter((item) => item);
+    if (codePoints.length == baseGlyphNames.length) {
+      glyphString = codePoints
+        .map((codePoint) => getCharFromCodePoint(codePoint))
+        .join("");
+
+      if (extension) {
+        const ZWJ = "\u200D";
+        switch (extension) {
+          case ".isol":
+            break;
+          case ".init":
+            glyphString = glyphString + ZWJ;
+            direction = "rtl";
+            break;
+          case ".medi":
+            glyphString = ZWJ + glyphString + ZWJ;
+            direction = "rtl";
+            break;
+          case ".fina":
+            glyphString = ZWJ + glyphString;
+            direction = "rtl";
+            break;
+          default:
+            break;
+        }
+      }
+    }
+  }
+
+  return [glyphString, direction];
```

Comment:
```suggestion
  return { glyphString, direction };
```
As mentioned in https://github.com/googlefonts/fontra/pull/2010#discussion_r2038957884